MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,306 @@
+
+{% endhighlight %}
+{% endtabs %}
+
+#### LostFocus
+
+If the validation mode is LostFocus, the value will be validated when the editor lost its focus. By default the [ValidationMode] is `LostFocus`
+
+#### PropertyChanged
+
+If the validation mode is PropertyChanged, The value will be validated immediately when it is changed.
+
+#### Manual
+
+The value should be validated manually by calling the [SfDataForm.Validate] or [SfDataForm.Validate(new List<string>())] method.
+
+The following code validates the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.Validate();
+
+{% endhighlight %}
+{% endtabs %}
+
+To validate the specific list of property value, pass the property names as argument.
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.Validate(new List<string>() {"FirstName", "Age" });
+
+{% endhighlight %}
+{% endtabs %}
+
+You can determine whether the data form or property is valid or not by using the `Validate` method.
+
+{% tabs %}
+{% highlight c# %}
+
+        bool isValid = this.dataForm.Validate();
+
+        List<string> strings = new List<string>();
+        strings.Add("FirstName");
+        bool isPropertyValid = this.dataForm.Validate(strings);
+
+{% endhighlight %}
+{% endtabs %}
+
+If the data form or property is valid, `true` will be returned. Or else `false` will be returned.
+
+#### Get Validation details through events
+
+You can get the validation details of the entire data form using the [ValidateForm] event of the data form.
+
+Note: This event will be raised once after the manual validation call using [SfDataForm.Validate()] method
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        ValidateForm="OnDataFormValidateForm">
+    </dataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.dataForm.ValidateForm += this.OnDataFormValidateForm;
+
+    private void OnDataFormValidateForm(object sender, DataFormValidateFormEventArgs e)
+    {
+        object dataObject = e.DataObject;
+        var errorMessage = e.ErrorMessage;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Valid message
+
+If the value meets the desired criteria, you can show the [ValidMessage]. As error message, the valid message will also be displayed at the bottom of the editor.
+
+{% tabs %}
+{% highlight c# %}
+
+    [DataFormDisplayOptions(ValidMessage = "Password strength is good")]
+    [Required(ErrorMessage = "Please enter the password")]
+    [RegularExpression(@"^(?=.*[a-z])(?=.*[A-Z])[a-zA-Z\d]{8,}$", ErrorMessage = "A minimum 8-character password should contain a combination of uppercase and lowercase letters.")]
+    public string Password { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customize error label text style
```

Comment:
Add this as last title in this page

## Validation label appearance customization

The data form supports to customize the style of both error and valid message label style easily. 

### Customize the error label text style
### Customize the valid message label text style

Reply:
content changed